build/pause/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# 3.1
+
+* The pause container gains a single handler to clean up orphaned zombie processes. ([#36853](https://prs.k8s.io/36853), [@verb](https://github.com/verb))
```

Comment:
typo: single->signal?

Reply:
Whoops, fixed. Thanks for catching that.